### ISSUE TITLE
Problems with loader=wait

### ISSUE THREAD

**Issue by rpontes** [malsup/cycle2#59]
When I use data-cycle-loader="wait" my expectation is respect the order.

But " the slides will always be displayed in their natural, markup order" doesn't work with me.

Please look my code:

https://gist.github.com/4320474.git

**Comment by malsup** [comment]
Thanks for reporting this!

**Comment by rpontes** [comment]
Doesn't work for me, look my generated code:

https://gist.github.com/4330207

gist:4330207

**Comment by malsup** [comment]
I messed up when uploading the changed file.  Please re-download.  Sorry for the trouble.

**Comment by rpontes** [comment]
I don't know why but doesn't work here.

Do I need put jquery.cycle2.js and jquery.cycle2.loader.js at page?

Here my code:

https://gist.github.com/4336222.git

**Comment by malsup** [comment]
You don't need to include the loader plugin, it's included in the main jquery.cycle2.js file.  You only need to make sure you have the latest cycle2 file.  Here's an example:

http://jquery.malsup.com/cycle2/demo/loader-wait.php

**Comment by malsup** [comment]
Fixed again!  See #65.

**Comment by aquasplash** [comment]
Download the latest version today.

jQuery v1.10.2
  jquery.cycle2.min.js
  jquery.cycle2.carousel.min.js
  jquery.cycle2.center.min.js

```
class="cycle-slideshow"
  data-cycle-loader="true"
  data-cycle-slides="li"
  data-cycle-paused="true"
  data-cycle-fx="carousel"
  data-cycle-carousel-visible="5"
```

The only problem with the loader.

Here, too, does not work.
http://jsfiddle.net/uXy6B
